django/cantusdb_project/main_app/templates/source_lists/source_list.html

```
@@ -158,6 +158,41 @@
+                        <input type="radio"
+                               class="form-check-input"
+                               id="inventoried-all"
+                               name="inventoried"
+                               value="all"
+                               {% if request.GET.inventoried != "inventoried" and request.GET.inventoried != "nonInventoried" %}checked{% endif %} />
+                        <label class="form-label form-check-label small" for="inventoried-all">
+                            All
+                        </label>
+                    </div>
+                    <div class="form-check form-check-inline">
+                        <input type="radio"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)